bin/templates/platform_www/cdv-electron-main.js

```diff
@@ -62,10 +89,27 @@ function createWindow () {
     });
 }
 
+function configureProtocol () {
+    protocol.registerFileProtocol(scheme, (request, _cb) => {
+        const url = request.url.substr(basePath.length + 1);
+        _cb({ path: path.normalize(`${__dirname}/${url}`) });
+    }, (error) => {
+        if (error) console.error('Failed to register protocol');
```

Comment:
Shouldn't the actual error message be logged, too?

Reply:
I have removed error method. It is deprecated.

> (node:53460) ProtocolDeprecateCallback: The callback argument of protocol module APIs is no longer needed.